{{ cookiecutter.package_name }}/.travis.yml

```diff
@@ -95,7 +91,7 @@ matrix:
 
         # Now try Astropy dev with the latest Python and LTS with Python 2.7 and 3.x.
         - os: linux
-          env: ASTROPY_VERSION=development
+          env: PYTHON_VERSION=3.7 ASTROPY_VERSION=development
```

Comment:
I would rather like python 3.7 to be the default in L30 above? or is that still premature?

Reply:
In Python release notes, they advised the more "conservative" users to wait till 3.7.1 for bugs to get sorted out. But that said, anyone creating a *new* package from this template might want 3.7 anyway? I'll leave that to your call.

Reply:
conservative users don't use the package template :trollface: 

I don't have strong preference either way, so I'm leaving this PR to be merged by @astrofrog.

Reply:
OK I don't have preference either way, so just lemme know if you want me to change it to 3.7 as global default. :smile: